src/test_l1_dsVideoDevice.c

```diff
@@ -522,15 +523,21 @@ void test_l1_dsVideoDevice_positive_dsGetDFC(void)
     UT_ASSERT_EQUAL((handle >= 0), true);
```

Comment:
for dsVideoDeviceInit & dsVideoDeviceTerm
Need to use `UT_ASSERT_EQUAL_FATAL` same as L2

Reply:
done